tests/prediction_pipeline/test_prediction_pipeline_tiled.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
We actually have tests for the tiled prediction already: https://github.com/bioimage-io/core-bioimage-io-python/blob/main/tests/test_prediction.py#L138
I don't mind refactoring these tests to the file here.